src/shared/bpf-dlopen.c

```diff
@@ -51,8 +51,8 @@ int dlopen_bpf(void) {
                         DLSYM_ARG(bpf_link__fd),
```

Comment:
(can't comment on lines out of diff range) should also look for libbpf.so.1 just above ^

It doesn't look like there is a way of providing multiple filenames, so will have to call this twice I guess...

Reply:
As mentioned in the other PR: use separate dlopen() + dlsym_many_or_warn() instead of dlopen_many_sym_or_warn()